src/error.rs

```diff
@@ -96,3 +96,86 @@ impl From<::std::io::Error> for StoreError {
         StoreError::IoError(e)
     }
 }
+
+#[derive(Debug, Fail)]
+pub enum MigrateError {
+    #[fail(display = "Database not found: {:?}", _0)]
```

Comment:
Nit: s/Database/database/?, as all other display messages are lowercase.

Reply:
Right, good to be consistent here. Done in bd584ac061d0896a86c5b2c4f42fe97be40d5d42.